peakina/helpers.py

```diff
@@ -39,15 +39,19 @@ class TypeInfos(NamedTuple):
 
 
 SUPPORTED_FILE_TYPES = {
-    "csv": TypeInfos(["text/csv", "text/tab-separated-values"], pd.read_csv),
+    "csv": TypeInfos(
+        ["text/csv", "text/tab-separated-values"],
+        read_csv,
+        ["preview", "chunksize"],
+    ),
     "excel": TypeInfos(
         [
             "application/vnd.ms-excel",
             "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
         ],
-        pd.read_excel,
+        read_excel,
         # these options are missing from read_excel signature in pandas 0.23:
-        ["keep_default_na", "encoding", "decimal"],
+        ["preview", "keep_default_na", "encoding", "decimal", "sheet_name"],
```

Comment:
This one also needs to be updated. It also `wraps()` `pd.read_csv` (and not `pd.read_excel()` anymore as we may think)

Reply:
👍🏾 Updated !